---
title: Errore del compilatore CS0255 | Documenti di Microsoft
ms.date: 2015-07-20
ms.prod: .net
ms.technology:
- devlang-csharp
ms.topic: article
f1_keywords:
- CS0255
dev_langs:
- CSharp
helpviewer_keywords:
- CS0255
ms.assetid: b45f5d5a-1923-4fe1-a858-e5ef5590a108
caps.latest.revision: 8
author: BillWagner
ms.author: wiwagn
translation.priority.ht:
- de-de
- es-es
- fr-fr
- it-it
- ja-jp
- ko-kr
- ru-ru
- zh-cn
- zh-tw
translation.priority.mt:
- cs-cz
- pl-pl
- pt-br
- tr-tr
translationtype: Machine Translation
ms.sourcegitcommit: a06bd2a17f1d6c7308fa6337c866c1ca2e7281c0
ms.openlocfilehash: aef8271bcff269ad6c0fe3f7464fff852782f583
ms.lasthandoff: 03/13/2017

---
# <a name="compiler-error-cs0255"></a>Errore del compilatore CS0255
stackalloc non può essere usato in un blocco catch o finally  
  
 Il [stackalloc](../../csharp/language-reference/keywords/stackalloc.md) parola chiave non può essere utilizzate un [catch](../../csharp/language-reference/keywords/try-catch.md) o [infine](../../csharp/language-reference/keywords/try-catch-finally.md) blocco. Per ulteriori informazioni, vedere [le eccezioni e gestione delle eccezioni](../../csharp/programming-guide/exceptions/index.md).  
  
 L'esempio seguente genera l'errore CS0255:  
  
```  
// CS0255.cs  
// compile with: /unsafe  
using System;  
  
public class TestTryFinally  
{  
   public static unsafe void Test()  
   {  
      int i = 123;  
      string s = "Some string";  
      object o = s;  
  
      try  
      {  
         // Conversion is not valid; o contains a string not an int  
         i = (int) o;  
      }  
  
      finally  
      {  
         Console.Write("i = {0}", i);  
         int* fib = stackalloc int[100];   // CS0255  
      }  
   }  
  
   public static void Main()  
   {  
   }  
}  
```
